src/TrendsOverview.svelte

```diff
@@ -1356,14 +1354,13 @@
           </p>
         </div>
         <div class="next-steps-item">
-          <h3>Analyze with covariates</h3>
+          <h3>Analyze with covariate data</h3>
           <p>
-            Analyze the data alongside other covariates in the COVID-19
-            Open-Data repository. COVID-19 Open-Data repo
+            Analyze this data alongside other covariates in the COVID-19 Open-Data repository.
           </p>
           <p>
             <a href="https://github.com/GoogleCloudPlatform/covid-19-open-data"
-              >COVID-19 Open Data Repository.</a
+              >Github repository</a
```

Comment:
Nit: Should this instead be "COVID-19 Open Data Github repository" to distinguish that project from the front-end repo?

On that note, should we more prominently link to the front-end repo?  The convention seems to be to have one of the github icons on the page and link there.

Reply:
I would go with COVID-19 Open Data Github repository or COVID-19 Open Data repository over Github Repository. I've pinged the copy writer to suggest using one of those options. I've also captured the link to Github issue.